### ISSUE TITLE
Position attribute for covers doesn't work with Bosch SHC and Velux

### ISSUE THREAD

**Issue by 23NH23** [hugobloem/stateful_scenes#64]
The position attribute in the cover domain doesn't work for the covers mentioned in the title. Unfortunately the attribute name is "current_position" instead "position" for both. Please add this attribute in a future version.

Thanks for that great integration.

Best regards,

Nils

**Comment by hugobloem** [comment]
You're right, seems like `position` isn't an attribute at all. I'll update this.
